certs.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"os"
+
+	"github.com/remind101/emp/Godeps/_workspace/src/github.com/bgentry/heroku-go"
+)
+
+var cmdCertAttach = &Command{
+	Run:      runCertAttach,
+	Usage:    "cert-attach",
+	NeedsApp: true,
+	Category: "certs",
+	Short:    "attach a certificate to an app",
+	Long:     `Attach a certificate to an app`,
```

Comment:
How about:

`Attaches an SSL key & certificate to an exposed web app's load balancer.`